### ISSUE TITLE
Latest commits not in this repository ?

### ISSUE THREAD

**Issue by aussiehash** [devlab21/simple-vimeo-downloader-extension#3]
Hello,

the last commits to this repo were from March, is there a newer branch ?

thanks

**Comment by ycherniavskyi** [comment]
@aussiehash actually not. Do you have any issue with this extension, that must be fixed?

**Comment by LoSunny** [comment]
Yes, the extension is not working, I've contacted through email about the issue
